src/serialization/portable/DefaultPortableWriter.ts

```diff
@@ -19,11 +19,11 @@ import {PortableSerializer} from './PortableSerializer';
 import {PositionalDataOutput} from '../Data';
 import {ClassDefinition, FieldDefinition} from './ClassDefinition';
 import {BitsUtil} from '../../util/BitsUtil';
-import {Portable, FieldType} from '../Portable';
+import {Portable, FieldType, PortableWriter} from '../Portable';
 import * as Long from 'long';
 
 /** @internal */
-export class DefaultPortableWriter {
+export class DefaultPortableWriter implements PortableWriter{
```

Comment:
Apart from deleting deprecation comments for classes, I noticed this one does not implement PortableWriter